truffle/src/main/ruby/core/rubinius/common/regexp.rb

```diff
@@ -128,6 +128,10 @@ def encoding
     source.encoding
   end
 
+  def casefold?
+    (options & IGNORECASE) > 0 ? true : false
+  end
+
```

Comment:
I don't know if anyone told you, but we're trying to keep to version 2.4.1 of Rubinius, just to be consistent. We'll probably upgrade later on.